engine/ladon/handler.go

```diff
@@ -290,8 +290,7 @@ func (e *Engine) Register(r *httprouter.Router) {
 }
 
 func (e *Engine) rolesList(ctx context.Context, r *http.Request, ps httprouter.Params) (*kstorage.ListRequest, error) {
-	var p kstorage.Roles
-
+	p := make([]kstorage.Role, 0)
```

Comment:
It is possible to make types that have an underlying type of a slice or map:

```suggestion
	p := make(kstorage.Roles, 0)
```